portfolio/src/main/java/com/google/sps/servlets/RequestServlet.java

```diff
@@ -13,26 +13,34 @@
 import javax.servlet.http.HttpServletResponse;
```

Comment:
I don't think the changes to this file are related to the new map functionality

Reply:
That is actually pretty wired that this shows up as an added change...I've always had this import here because the doGet and doPost functions don't work without this